docs/content/configuration-and-management/maas-controller-overview.md

```
@@ -1,346 +1,225 @@
+    For detailed information about controller architecture, reconciliation flow, and authentication internals, see:
+    
+    - [Controller Architecture](../architecture-internals/controller-architecture.md)
+    - [Reconciliation Flow](../architecture-internals/reconciliation-flow.md)
+    - [Authentication Internals](../architecture-internals/authentication-internals.md)
 
 ---
 
-## 2. High-Level Architecture
+## Deployment and Prerequisites
 
 ```mermaid
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)